company.el

```diff
@@ -1366,6 +1373,9 @@ update if FORCE-UPDATE."
   (when (or force-update (not (equal selection company-selection)))
     (setq company-selection selection
           company-selection-changed t)
+    (when company-auto-update-doc
```

Comment:
Do you still need this call here? What happens if you remove it?

Reply:
Oh, no! This has just accidentally crept back in when I moved from elpa sources to a git fork.  I'll force-push a new version soon.  I'm sorry.